README.md

```diff
@@ -2,15 +2,6 @@
 
 [Latest Draft](https://w3c-ccg.github.io/ldp-bbs2020)
 
-The following repository contains the specification for a [linked data signature](https://w3c-ccg.github.io/ld-proofs/) using the [BBS+ Signature Algorithm](https://eprint.iacr.org/2009/095.pdf) using [BLS12-381](https://tools.ietf.org/id/draft-yonezawa-pairing-friendly-curves-00.html#rfc.section.2.4) elliptic curve based key pairs.
+The following repository contains the specification for a [linked data signature](https://w3c-ccg.github.io/ld-proofs/) using the [BBS+ Signature Algorithm](https://mattrglobal.github.io/bbs-signatures-spec/) using [BLS12-381](https://tools.ietf.org/id/draft-yonezawa-pairing-friendly-curves-00.html#rfc.section.2.4) elliptic curve based key pairs.
```

Comment:
Do you retain the link to https://eprint.iacr.org/2009/095.pdf in the `bbs-signatures-spec` ? I searched but could not find it.

Reply:
Will file an issue on the spec to ensure this is maintained